### ISSUE TITLE
compilation error when using nake

### ISSUE THREAD

**Issue by kobi2187** [yglukhov/nimx#347]
following the instructions on "building for different platforms",
I get the following error:
...
```
Hint: plists [Processing]
Hint: xmltree [Processing]
Hint: xmlparser [Processing]
Hint: parsexml [Processing]
..\..\..\.nimble\pkgs\nimx-0.1\nimx\naketools.nim(188, 30) Error: undeclared identifier: 'TimeInfo'
```
same with head:
```..\..\..\.nimble\pkgs\nimx-#head\nimx\naketools.nim(188, 30) Error: undeclared identifier: 'TimeInfo'```

probably some library is missing? or some dependency is not updated on my end.

**Comment by kobi2187** [comment]
oh, it was just replaced with DateTime

**Comment by Tormund** [comment]
fixed in #348 pr
